renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -1097,6 +1097,9 @@ void WrappedOpenGL::DeleteContext(void *contextHandle)
       m_Real.glDeleteTextures(1, &ctxdata.GlyphTexture);
   }
 
+  if(ctxdata.m_ClientMemoryVBOs[0])
+    m_Real.glDeleteBuffers(ARRAY_COUNT(ctxdata.m_ClientMemoryVBOs), ctxdata.m_ClientMemoryVBOs);
```

Comment:
This should go through our `glDeleteBuffers` function, so that we can remove our own internal representations as well (to correspond with the `glGenBuffers` going through our function). Otherwise it still hits the assert for leaking resource records.